### ISSUE TITLE
Listing collection entries in Control Panel doesn't work over https

### ISSUE THREAD

**Issue by wojtiku** [statamic/v2-hub#910]
### Steps to reproduce
1. Deploy your site to https
2. Visit a collection listing
3. AJAX request is blocked due to mixed content - the script requests http url instead of https
### Expected behaviour

Should go to https and just work.
### Actual behaviour

Goes to http and is blocked.
### Server

Apache/2.4.7 (Ubuntu)
PHP 5.5.9

This is blocking releasing a site for me :/
## Temporary workaround

In the `boot` method of `/statamic/core/Providers/AppServiceProvider.php` add the following code:

```
$this->app['request']->server->set('HTTPS', $this->app->environment() != 'dev');
```

It sets the request protocol to HTTPS on every environment but `dev`.

**Comment by wojtiku** [comment]
Ok, so after some digging and talking to our sysadmin it turns out there is a twist - our site is behind a reverse proxy - so nginix contacts apache before serving the site and it contacts it via http. In our case the real protocol is in the `X-Forwarded-Proto` HTTP header. So

But the real question for me is - why can't the code just print urls starting with slash? This way it would just work, for everyone. Hostname and protocol aren't needed at all here.

**Comment by jackmcdade** [comment]
I agree. The CP should use relative URLs.

**Comment by jasonvarga** [comment]
👍 Laravel generates absolute urls by default. We can adjust that.

**Comment by jackmcdade** [comment]
This got fixed up in #1140 and will be in the next release. Closing this issue to consolidate!
